plugins/woocommerce-admin/client/marketplace/components/likert-scale/likert-scale.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './likert-scale.scss';
+
+export interface LikertScaleProps {
+	title: string;
+	fieldName: string;
+	onValueChange: ( value: number ) => void;
+	validationFailed?: boolean;
+}
+
+export interface LikertChangeEvent {
+	target: {
+		value: number;
+	};
+}
+
+export default function LikertScale( props: LikertScaleProps ): JSX.Element {
+	const { title, fieldName, onValueChange, validationFailed } = props;
+	const scaleOptions = [
+		{ value: 1, emoji: '😔', label: __( 'Strongly disagree' ) },
+		{ value: 2, emoji: '🙁', label: __( 'Disagree' ) },
+		{ value: 3, emoji: '😐', label: __( 'Neutral' ) },
+		{ value: 4, emoji: '🙂', label: __( 'Agree' ) },
+		{ value: 5, emoji: '😍', label: __( 'Strongly agree' ) },
+	];
+
+	const classes = classnames( 'woocommerce-marketplace__likert-scale', {
+		'validation-failed': validationFailed,
+	} );
+
+	function valueChanged( e: React.ChangeEvent<HTMLInputElement> ){
```

Comment:
Prettier demands whitespace.

```suggestion
	function valueChanged( e: React.ChangeEvent< HTMLInputElement > ) {
```